astro_2.0/components/navigation/NavButton/NavButton.tsx

```diff
@@ -40,7 +44,15 @@ export const NavButton: VFC<NavButtonProps> = props => {
 
   const [showLoader, setShowLoader] = useState(false);
 
-  const isActive = useIsHrefActive(href);
+  const { asPath } = router;
+  const pathParts = asPath.split('/').filter(item => item !== '');
+  const daoId = pathParts[0] === 'dao' ? pathParts[1] : '';
```

Comment:
We can get active dao id from url just from router.query object.
const daoId = router.query.dao

Reply:
fixed